concent-vm/configure.yml

```diff
@@ -0,0 +1,108 @@
+          include_tasks: ../concent-builder/common_tasks/prerequire-configure-tasks.yml
+
+        - name:   Install basic packages
+          apt:  name={{ item }}
+          with_items:
+            - ncdu
+            - tree
+            - htop
+            - tmux
+            - vim
+
+        - name:   Install dependencies
+          apt:  name={{ item }}
+          with_items:
+            - build-essential
+            - python2.7
+            - python-pip  # For Python 2
+            - libssl-dev
+            - pkg-config
+            - libsecp256k1-dev
+            - git
+            - docker-engine={{ docker_version }}
+            - ansible
+            - libpq-dev
+            - gcc
+            - python3-yaml
+            - postgresql-{{ postgres_version }}
+            - python-psycopg2
+            - blender
+            - libsm6
+
+        - name:   Create user that will be used for building stuff
+          user:
+            name:   builder
+            state:  present
+            groups: docker
+
+        - name:    Add user 'vagrant' to the 'docker' group
+          user:
+            name:   vagrant
+            groups: docker
+            append: yes
+
+        - name:   include common postrequire tasks from `postrequire-configure-tasks.yml` file
+          include_tasks: ../concent-builder/common_tasks/postrequire-configure-tasks.yml
+
+    - become:      yes
+      become_user: builder
+      block:
+        - name:   include start docker service tasks from `start-docker-services.yml` file
+          include_tasks: start-docker-services.yml
+
+        - name:   include building nginx tasks from `build-nginx-storage.yml` file
+          include_tasks: build-nginx-storage.yml
+
+    - become:      yes
+      become_user: root
+      block:
+        - name:   Configure PostgreSQL to accept local connection
+          lineinfile:
+            path: "/etc/postgresql/{{ postgres_version }}/main/postgresql.conf"
+            line: "listen_addresses='*'"
+
+        - name:   Configure postgresql to trust local connection without password
+          copy:
+            src:        "{{ concent_deployment_dir }}/concent-vm/pg_hba.conf"
+            dest:       "/etc/postgresql/{{ postgres_version }}/main/pg_hba.conf"
+            remote_src: yes
+            owner:      builder
+            group:      builder
+
+        - name:   Restart postgresql service
+          service:
+            name:  postgresql
+            state: restarted
+
+        - name:   Copy nginx-storage.service which is run nginx-storage locally
+          copy:
+            src:        "{{ concent_deployment_dir }}/concent-vm/nginx-storage.service"
+            dest:       "/etc/systemd/system/nginx-storage.service"
+            remote_src: yes
+            owner:      builder
+            group:      builder
+
+        - name:   Run nginx-storage service
+          service:
+            name:  nginx-storage
+            state: started
```

Comment:
You should use `enabled: yes` to make the service run at startup.

Reply:
Done.

Reply:
Does not seem done yet. Have you pushed the latest code?

Reply:
Done.